source/extensions/common/proxy_protocol/proxy_protocol_header.h

```diff
@@ -61,6 +62,11 @@ void generateProxyProtoHeader(const envoy::config::core::v3::ProxyProtocolConfig
 // Generates the v2 PROXY protocol local command header and adds it to the specified buffer
 void generateV2LocalHeader(Buffer::Instance& out);
 
+// Generates the v2 PROXY protocol header and the TLV vector into the specified buffer.
+// TODO: fix const.
```

Comment:
[q] what's this issue with `const` here?

Reply:
Ah. `ProxyProtocolData` is const in many places but TLVs are mutable. Just reminding myself this should be consistent.